tests/tcl/tests/test_helper.tcl

```diff
@@ -56,7 +56,6 @@ set ::all_tests {
     unit/introspection
     unit/limits
     unit/geo
-    unit/command
```

Comment:
If we delete it in the migration procedure, developers may need a two phase testing in their local:
1. run tcl tests
2. run go tests

Any idea to simplify the test executing?

Reply:
If we don't delete it, the situation doesn't change. If only the migration can be eventually closed, I don't think it's a big deal.

Since we do have TCL tests and go IT cases, we always run both. There's no way to avoid it. Only you may batch them in one script call but it seems not necessary.

Reply:
Sounds good to me.